node/actors/sync_blocks/src/peers/mod.rs

```diff
@@ -237,7 +257,7 @@ impl PeerStates {
         Ok(last_contiguous_stored_block)
     }
 
-    fn validate_sync_state(&self, state: SyncState) -> anyhow::Result<BlockNumber> {
+    fn validate_sync_state(&self, state: SyncState) -> anyhow::Result<(BlockNumber, BlockNumber)> {
```

Comment:
can we have some "block range" type here instead of tuple?

Reply:
+1